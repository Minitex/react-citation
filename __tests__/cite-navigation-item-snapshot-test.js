import React from 'react'
import renderer from 'react-test-renderer'
import CiteNavigationItem from '../src/cite-navigation-item'

describe('CiteNavigation Tests', () => {
  const item = {class_name: "active", text: "Foo"}

  it("renders a navigation item", function(){  
    const component = renderer.create(
      <CiteNavigationItem label="Foo" key="0" class_name="active" setActiveItem={() => {}} />
    )
    let tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})