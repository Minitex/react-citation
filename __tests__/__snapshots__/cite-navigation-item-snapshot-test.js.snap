exports[`CiteNavigation Tests renders a navigation item 1`] = `
<li
  className={undefined}
  role="presentation">
  <a
    href="#"
    onClick={[Function]}>
    Foo
  </a>
</li>
`;
